kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/index.vue

```diff
@@ -89,13 +96,11 @@
     },
     vuex: {
       getters: {
-        pageState: state => state.pageState,
-        // fake date for page-status
-        contentName: () => 'Adding Fractions',
-        userName: () => 'James Howard',
-        progress: () => 1,
-        assessment: () => '4 of 5',
-        date: () => '18 Nov 2016',
+        // pageState: state => state.pageState,
+        channelId: state => state.pageState.channelId,
+        attemptLogs: state => state.pageState.attemptLogs,
+        exercise: state => state.pageState.exercise,
+        userName: state => state.core.session.username,
```

Comment:
This getter is incorrect, btw - this should be the user's name, not the currently logged in user's name :)